packages/frontend-api/src/Resources/config/services.yml

```diff
@@ -0,0 +1,16 @@
+services:
+    _defaults:
+        autowire: true
+        autoconfigure: true
+        public: false
+
+    Shopsys\FrontendApiBundle\:
+        resource: '../../**/*{Resolver}.php'
+
+    Shopsys\FrontendApiBundle\Routing\EnabledOnCurrentDomainChecker: ~
```

Comment:
Good catch, thanks!

Reply:
Well, `%shopsys_frontend_api.enabled_domains%` is not registered as parameter. 
Into this service it's passed in `ShopsysFrontendApiExtension.php`

Reply:
Okay that is something I do not like, because when you are looking for arguments, that are wired to the service, you look into services.yml and you find nothing, then you need to start thinking, where it might be.

Reply:
I rewrote it to the standard parameter in a14a795771f082c8dbbc0d18cf4d3cc0d33eee39 and 19baa4e863e844360de88dd38dffe3d6e0d301fa
(documentation and upgrade notes are changed too in 62c5a3973b1c1d78372ed2a01da6336b1580bffe and ee698a5e1521a7dea930e907e14869dd9fa67a56)